src/gridfs/upload.ts

```diff
@@ -556,7 +562,7 @@ function writeRemnant(stream: GridFSBucketWriteStream, callback?: Callback): boo
 function checkAborted(stream: GridFSBucketWriteStream, callback?: Callback<void>): boolean {
   if (stream.state.aborted) {
     if (typeof callback === 'function') {
-      callback(new MongoDriverError('this stream has been aborted'));
+      callback(new MongoStreamClosedError('Stream has been aborted'));
```

Comment:
In the same vein as [this comment on NODE-3421](https://github.com/mongodb/node-mongodb-native/pull/2910#pullrequestreview-712206125) we may want to consider adding a default message for MongoStreamClosedError with an optional message parameter for cases where more information is needed